examples/amqp/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>cloudevents-examples</artifactId>
+    <groupId>io.cloudevents</groupId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>cloudevents-amqp-vertx</artifactId>
+
+  <dependencies>
+  	 <dependency>
+  	    <groupId>io.cloudevents</groupId>
+  	    <artifactId>cloudevents-amqp-proton</artifactId>
+  	    <version>${project.version}</version>
+  	 </dependency>
+     <dependency>
+     	<groupId>io.vertx</groupId>
+     	<artifactId>vertx-core</artifactId>
+     	<version>3.9.3</version>
```

Comment:
Can you use vert.x 4? If not, no problems.

Also can you make the vertx version a property?

Reply:
sure, i can use vert.x 4. the onSuccess call is nice 👍